src/components/RestaurantDetails/components/Filters/components/Favourite/Favourite.js

```diff
@@ -0,0 +1,18 @@
+//Libraries
+import React from 'react';
+
+//Icons
+import heartIcon from './images/heart.svg';
+
+//CSS
+import './Favourite.css';
+function Favourite() {
```

Comment:
instead of creating component Favourite create a separate common component, which will take image and label and will render it, by naming this component Favourite, this component is not reusable